substratevm/src/com.oracle.svm.driver/src/com/oracle/svm/driver/NativeImage.java

```diff
@@ -97,6 +97,10 @@
 import com.oracle.svm.util.ModuleSupport;
 import com.oracle.svm.util.ReflectionUtil;
 
+import static com.oracle.svm.driver.launcher.ContainerSupport.replaceContainerPaths;
```

Comment:
We try to avoid static imports if possible.

Most of the time it is better to use `ContainerSupport.mountMappingFor` and so on. While `replaceContainerPaths` has a clear meaning on its on, clearly it's not so straightforward for `TargetPath`.

Also if you avoid using `import static` here you can rename `replaceContainerPaths` to `replacePaths` because below it will always show up as `ContainerSupport.replacePaths` thus it is easy to understand what it really does.

Reply:
Addressed in commit https://github.com/oracle/graal/pull/6496/commits/e22646ba54ab57f8bd11e56b1a805c7ef3222bce